clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -502,6 +502,10 @@ def err_sls_hardening_arm_not_supported : Error<
 def warn_drv_large_data_threshold_invalid_code_model: Warning<
   "'%0' only applies to medium and large code models">,
   InGroup<UnusedCommandLineArgument>;
+def warn_drv_math_errno_reenabled_after_veclib: Warning<
+  "math errno re-enabled by '%0' after it was implicitly disabled by '%1',"
+  " this may prevent vectorization with the specified vector library">,
+  InGroup<MathErrnoEnabledWithVecLib>;
```

Comment:
I'd rather not link the warning to any specific optimisation.  I'd prefer something simpler like "..., this may limit the utilisation of the vector library.".

Reply:
Done :+1: